woof-code/rootfs-petbuilds/Xdialog/petbuild

```diff
@@ -1,12 +1,19 @@
 download() {
-    [ -f Xdialog-2020.10.tar.bz2 ] || wget -t 3 -T 60 http://distro.ibiblio.org/puppylinux/sources/x/Xdialog-2020.10.tar.bz2
+    if [ ! -f Xdialog-14c5747051bd196c.tar ]; then
+        fossil open --workdir Xdialog_2022-01-03_013714_14c5747051 http://chiselapp.com/user/jamesbond/repository/xdialog 14c5747051 
```

Comment:
Ok, depends on `fossil`.

Reply:
Yes, I don't trust the Fossil->Git conversion, and don't want to repeat this process for every Xdialog commit